supertokens_python/recipe/emailpassword/syncio/__init__.py

```diff
@@ -24,10 +24,15 @@ def update_email_or_password(
     email: Union[str, None] = None,
     password: Union[str, None] = None,
     user_context: Union[None, Dict[str, Any]] = None,
+    apply_password_policy: Union[bool, None] = None,
```

Comment:
apply_password_policy should be second last - user_context should be the last one